ViewControllers/ViewController.swift

```diff
@@ -40,13 +40,14 @@ class ViewController: UIViewController {
 	override func viewWillAppear(_ animated: Bool) {
 		super.viewWillAppear(animated)
 		self.navigationController?.navigationBar.barStyle = .default
-		self.navigationController?.navigationBar.tintColor = .black
-		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.black]
+		self.navigationController?.navigationBar.tintColor = .white
+        self.navigationController?.navigationBar.barTintColor = .systemBlue
+		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
 	}
 	
 	func authorizeWith(account: String, onCompletion:@escaping(Bool)->()) {
 		authorizedAccount = account
-		let apiKey = "YOUR_API_KEY"
+		let apiKey = "5d59cb10-66cb-405b-ab54-b4d48132f383"
```

Comment:
Revert this change, since the drone repo (public) should not contain credentials.

Reply:
updated